src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/staff/RequestStaffCalculationService.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.service.staff
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsworkload.client.CommunityApiClient
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.GetWeeklyHours
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.SuccessUpdater
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.reduction.GetReductionService
+
+@Service
+class RequestStaffCalculationService(
+  private val weeklyHours: GetWeeklyHours,
+  private val getReductionService: GetReductionService,
+  @Qualifier("communityApiClient") private val communityApiClient: CommunityApiClient,
+  private val successUpdater: SuccessUpdater
+) {
+
+  fun requestStaffCalculation(staffCode: String, teamCode: String) {
+    communityApiClient.getStaffSummaryByCode(staffCode)
+      .map { it.grade }
+      .map { staffGrade ->
+        val weeklyHours = weeklyHours.findWeeklyHours(staffCode, teamCode, staffGrade)
```

Comment:
Just a thought - I suppose this is temporary, but it is a duplicate of the code in WorkloadCalculationService.calculate - might it be worth creating an abstraction? Probably not.